cs/v1/CS_v1.0.0.yaml

```diff
@@ -252,6 +266,21 @@ paths:
           description: The date since when the estimated arrival and departures of vessels in a given port is required
           required: true
         - $ref: ../../models/CommonComponents.yaml#/components/parameters/APIVersionMajor
```

Comment:
I thought we agreed not to link outside the API? Please copy the `APIVersionMajor` into this API and link to it (we will let Spectral handle the validation